### PR TITLE
rmt-cli repos clean followup

### PR DESCRIPTION
**Pull request by tmuntaner**
## Description

Followup from the `rmt-cli repos clean` demo.

* Added missing bash completion.
* Add clean message to products disable.
* Show estimated clean size.

## Change Type

*Please select the correct option.*

- [x] **Bug Fix** (a non-breaking change which fixes an issue)

## Checklist

*Please check off each item if the requirement is met.*

- [x] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [x] I have provided comments for any hard-to-understand code.
- [x] I have documented the `MANUAL.md` file with any changes to the user experience.
- [x] RMT's test coverage remains at 100%.
- [x] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

### CODE REVIEW

File: spec/lib/rmt/cli/products_spec.rb [hunk 1]
```diff
@@ -380,6 +380,7 @@
       output += "Disabling #{product.friendly_name}:\n"
       output += "  #{product.friendly_name}:\n"
       repos.pluck(:name).sort.each { |repo| output += "    Disabled repository #{repo}.\n" } unless repos.empty?
+      output += "\n\e[1mTo clean up downloaded files, please run 'rmt-cli repos clean'\e[22m\n"
```
Comment (lagartoflojo) [inline spec/lib/rmt/cli/products_spec.rb:383]: We shouldn't use "please", as per our [docu guidelines](https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html#sec-vocabulary):

```suggestion
      output += "\n\e[1mTo clean up downloaded files, run 'rmt-cli repos clean'\e[22m\n"
```

File: spec/lib/rmt/cli/products_spec.rb [hunk 2]
```diff
@@ -392,6 +393,7 @@
         output += "Disabling #{product.friendly_name}:\n"
         output += "  #{product.friendly_name}:\n"
         output += "    All repositories have already been disabled.\n"
+        output += "\n\e[1mTo clean up downloaded files, please run 'rmt-cli repos clean'\e[22m\n"
```
Comment (lagartoflojo) [inline spec/lib/rmt/cli/products_spec.rb:396]:
```suggestion
        output += "\n\e[1mTo clean up downloaded files, run 'rmt-cli repos clean'\e[22m\n"
```

File: lib/rmt/cli/products.rb [hunk 3]
```diff
@@ -67,6 +67,8 @@ def enable(*targets)
   REPOS
   def disable(*targets)
     change_products(targets, false, false)
+
+    puts "\n\e[1m" + _("To clean up downloaded files, please run '%{command}'") % { command: 'rmt-cli repos clean' } + "\e[22m"
```
Comment (lagartoflojo) [inline lib/rmt/cli/products.rb:71]:
```suggestion
    puts "\n\e[1m" + _("To clean up downloaded files, run '%{command}'") % { command: 'rmt-cli repos clean' } + "\e[22m"
```

**Review by lagartoflojo — COMMENTED** [review]
I noticed in the original PR #595 that we didn't add a changelog entry for this feature. Can we do this as part of this PR?

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/129168/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 1850788c4bc7b45ca7b1216a141f86174d7e3b97 into 892ab45071aad0523d98736422cdc383f80fa824)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/129169/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 6b7601047b166849b8b21b7acc4278b6db229349 into c23f1781426bb1b7778e54c87770768dfe903264)

**Comment by lcaparroz** [comment]
> I noticed in the original PR #595 that we didn't add a changelog entry for this feature. Can we do this as part of this PR?

@tmuntaner added the changelog in another PR between #595 and this one, you can check commit 6ded5bf.

**Comment by lagartoflojo** [comment]
Great, thanks!

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/129170/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 6b7601047b166849b8b21b7acc4278b6db229349 into fef6b1e9649e209eddfab9923dc5ba9e11ed909a)

File: lib/rmt/cli/products.rb (hunk 3 above)
Comment (lcaparroz) [reply lib/rmt/cli/products.rb]: Fixed it in 0e54832 :heavy_check_mark:

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/129171/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 6a983a8581c050a9b7de61dc72ff09102bae0d96 into f34602cb37c2b137cc71df1d58c92af6db2cc21e)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/129665/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 6a983a8581c050a9b7de61dc72ff09102bae0d96 into f34602cb37c2b137cc71df1d58c92af6db2cc21e)

**Review by wstephenson — APPROVED** [review]
lgtm

**Pull request merged**